edx_django_utils/cookie_consent/README.rst

```diff
@@ -0,0 +1,25 @@
+Cookie consent
+==============
+
+Utilities for respecting user consent regarding different cookie categories.
+
+"Cookie" is a shorthand here for various types of tracking and storage, but cookies are the main focus.
+
+Configuration
+-------------
+
+Set ``COOKIE_CONSENT_SOURCE`` to the module and name of a subclass of ``ConsentSource``, either one provided by edx-django-utils or from another package. This class will be responsible for answering whether the user has consented to certain categories of cookies. Known values:
+
+- ``edx_django_utils.cookie_consent.OnlyNecessary`` is the default and will only permit strictly necessary cookies.
+- ``edx_django_utils.cookie_consent.OneTrustSource`` will read and interpret the cookie set by OneTrust's consent UI.
```

Comment:
Again, ok as an example, but it should be from a different library.

Reply:
In general terms, edx-django-utils shouldn't have to know about specific consent cookie formats, even of the most popular solutions like OneTrust. But on the other hand, I'm not sure I can justify making a Python package that contains about 20 lines of code.

Reply:
I misread something and thought we were adding some sort of OneTrust dependency. I think this is probably fine. Maybe it could be in its own module (e.g. `cookie_consent.sources.onetrust`)? This is just a thought. Take it or leave it. :)